packages/react-vapor/src/components/form/Form.tsx

```diff
@@ -0,0 +1,32 @@
+import * as classNames from 'classnames';
+import * as React from 'react';
+
+type FormMods = 'mod-header-padding';
+
+export interface IFormProps {
+    title?: string;
+    className?: string | string[];
+    children?: React.ReactNode;
+    asCard?: boolean;
```

Comment:
shouldn't `asCard` be a mod?

Reply:
I was thinking of keeping only the `mod-X` classes as actual mods. Since `card` is `material-card`, I left it out :P

If you like the `mods` idea and think putting `material-card` in there is better, I would be OK with changing it :P